scripts/gcovr

```diff
@@ -526,7 +526,8 @@ def process_gcov_data(data_fname, covdata, source_fname, options):
         print('  currdir      '+currdir)
         print('  gcov_fname   '+data_fname)
         print('               '+str(segments))
-        print('  source_fname '+source_fname)
+        if not source_fname is None:
+            print('  source_fname '+source_fname)
```

Comment:
PEP-8 [suggests](https://www.python.org/dev/peps/pep-0008/#programming-recommendations) to write the conditional like `if source_fname is not None`. Let's use that for consistency.

Also, do you think `print('  source_fname '+(source_fname if source_fname is not None else '(None)'))` might be better? This makes it easier to see that the output is aligned, and makes it clearer that this field is explicitly empty. Or if we simply concatenate `str(source_fname)` rather than `source_fname`, we don't even need any conditionals.

Reply:
I went with `str(source_fname)` as suggested.